### PR TITLE
Add Observatorium API manifests

### PR DESCRIPTION
**Pull request by kakkoyun**
cc @brancz

### CODE REVIEW

**Comment by kakkoyun** [comment]
@metalmatze how about now?

File: environments/openshift/observatorium.libsonnet [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  apiVersion: 'v1',
+  kind: 'Template',
+  metadata: {
+    name: 'observatorium',
+  },
+  objects: [
+    $.observatorium.api[name]
+    for name in std.objectFields($.observatorium.api)
+  ],
+  parameters: [
```
Comment (squat) [inline environments/openshift/observatorium.libsonnet]: this part about adding the parameters is specific to the templating part; should this go in the observatorium.jsonnet, as that file is for generating the template?

Comment (kakkoyun) [reply environments/openshift/observatorium.libsonnet]: Yes, actually better idea.

File: environments/openshift/manifests/observatorium-api-template.yaml [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      port: 8080
+      targetPort: 8080
+    - name: https
+      port: 8081
+      targetPort: https
+    selector:
+      app.kubernetes.io/name: observatorium-api
+parameters:
+- name: NAMESPACE
+  value: telemeter
```
Comment (metalmatze) [inline environments/openshift/manifests/observatorium-api-template.yaml]: I don't think this namespace exists anywhere, but it will be overwritten anyway.

**Pull request merged**